### ISSUE TITLE
Fix analog vote > general abstain/yes/no

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#2583]
**Description:**
if an analog poll is created you can activate general abstain/yes/no, but you cannot give values for these options, making them obsolete.

**Recreation:** 
1. navigate into election
2. start creating an analog election
3. activate general abstain (or another)
4. no additional field is created where you could give the information how many people used this function

**Wanted behaviour:** 
If general abstain/yes/no is activated in an analog election then a text field where you can give the amount of people who generally abstained.

**Comment by reiterl** [comment]
The action `poll.create` seems to have missing optional properties.  Waiting for the linked backend issue.

**Comment by rrenkert** [comment]
@bastianjoel somwething for list in #6152?

**Comment by bastianjoel** [comment]
Yes. Analog poll form will be rewritten completely so we should take this into account.

**Comment by bastianjoel** [comment]
This is also part of #2714. Therefore closing.
